src/d3a/models/myco_matcher/pay_as_bid.py

```diff
@@ -0,0 +1,45 @@
+from d3a.constants import FLOATING_POINT_TOLERANCE
+
+
+class PayAsBidMatch:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def sorting(obj, reverse_order=False):
+        if reverse_order:
+            # Sorted bids in descending order
+            return list(reversed(sorted(
+                obj.values(),
+                key=lambda b: b.energy_rate)))
+
+        else:
+            # Sorted bids in ascending order
+            return list(sorted(
+                obj.values(),
+                key=lambda b: b.energy_rate))
+
```

Comment:
This method seems to be an exact duplicate of [the one defined in `MycoMatcher`](https://github.com/gridsingularity/d3a/blob/2f5abe0b27603a78e6a8ae5d79587a729e913059/src/d3a/models/myco_matcher/__init__.py#L14-L26). I would suggest refactoring it and using one common implementation.